intro/journaling_api.md

```diff
@@ -147,27 +425,197 @@ Your call results in a response containing a JSON object listing all the events
+
+The validate link is provided alongside every `204 No Content` response, and a simple GET request looks like this:
+
+```
+curl -X GET \
+  https://api.adobe.io/events/organizations/23294/integrations/54108/f89067f2-0d50-4bb2-bf78-209d0eacb6eb/validate?since=moose:e7ba778b-dace-4994-96e7-da80e7125233:2159b72c-e284-4899-b572-08da250e3614 \
+  -H "Authorization: Bearer $USER_TOKEN" \
+  -H "x-api-key: $API_KEY"
+```
+
+```
+HTTP/1.1 400 Bad Request
+
+Position moose:e7ba778b-dace-4994-96e7-da80e7125233:2159b72c-e284-4899-b572-08da250e3614 may or may not have existed but it surely doesn't exist anymore.
+```
+
+The above response corresponds to validating a positon that had expired. In case, the positon actually corresponded to the end of the journal, we could get a `204 No Content` or a `200 OK` (with events `since` the position) response back from the validate API. 
+
+```
+curl -X GET \
+  https://api.adobe.io/events/organizations/23294/integrations/54108/f89067f2-0d50-4bb2-bf78-209d0eacb6eb/validate?since=penguin:41322b44-c2e9-4b44-8354-ba2173064d24:752f6e67-d7e4-48d3-9f51-452936268fbb \
+  -H "Authorization: Bearer $USER_TOKEN" \
+  -H "x-api-key: $API_KEY"
+```
+
+```
+HTTP/1.1 204 No Content
+Link: </events/organizations/23294/integrations/54108/f89067f2-0d50-4bb2-bf78-209d0eacb6eb?since=penguin:41322b44-c2e9-4b44-8354-ba2173064d24:752f6e67-d7e4-48d3-9f51-452936268fbb>; rel="next"
+
+```
+
+Notice the lack of the validate link in the repsonse. The success http response code signifies the position is valid. `204 No Content` signifies that your application is at the end of the journal.
+
+Note: Validating a position is an expensive affair and it should not be a part of regular event consumption. The only place for the validation link to be called is when your client application comes online and tries to read events from a position that was persisted previously. In case the Journaling API returns a `204 No Content` response for a previously persisted position - your application could check whether the position has expired by calling the validate link.
+
+In case, the position has indeed expired, your application now needs to reset the position in the journal. You could chose to start consuming events from the `oldest available position` or the `latest` position depending on your use case.
+
+### No Events in Journal
+
+If in case, the Journaling API when called without any query parameters responds with a `204 No Content` response it signifies that there aren't actually any events in the journal. Either there may have been events in the past that have now expired, or, there never were any events in the journal.
```

Comment:
s/If in case, the Journaling API when called without any query parameters responds/If the Journaling API, when called without any query parameters, responds/